core/dbt/contracts/rpc.py

```diff
@@ -63,6 +63,16 @@ class RPCCompileParameters(RPCParameters):
     state: Optional[str] = None
 
 
+@dataclass
+class RPCListParameters(RPCParameters):
+    resource_types: Optional[List[str]] = None
+    models: Union[None, str, List[str]] = None
+    exclude: Union[None, str, List[str]] = None
+    select: Optional[List[str]] = None
```

Comment:
I believe the type of `select` should be `Union[None, str, List[str]]` as well. It should work just the same as `models` and `exclude`.

Reply:
Fixed!